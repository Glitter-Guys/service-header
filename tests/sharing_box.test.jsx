// Link.react.test.js
import React from 'react';
import SharingBox from '../client/src/components/sharing_box';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';




test('SharingBox fits the snapshot', () => {
  const component = renderer.create(
    <SharingBox text='SharingBox' ></SharingBox>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

// const checkbox = shallow(<SharingBox text='SharingBox' />);

// expect(checkbox.text()).toEqual('SharingBox');

});

test('SharingBox says SharingBox when rendered', () => {

  
  const sharingBox = shallow(<SharingBox text='SharingBox' />);
  
  expect(sharingBox.text()).toEqual('SharingBox');
  
  });