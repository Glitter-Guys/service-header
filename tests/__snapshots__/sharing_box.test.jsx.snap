// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SharingBox fits the snapshot 1`] = `
<div
  className="sharing-box"
>
  <p>
    SharingBox
  </p>
</div>
`;
